src/core/libraries/Errors.sol

```diff
@@ -18,142 +18,140 @@ library Errors {
     error DelegateCall();
 
     /*//////////////////////////////////////////////////////////////////////////
-                                 SABLIER-V2-LOCKUP
+                                 SABLIER-LOCKUP
```

Comment:
not centered

we are using [this](https://github.com/transmissions11/headers) for the headers, but with 76 spaces instead of 64

Reply:
Thanks for the tip.